tst/EnergyPlus/unit/SolarShading.unit.cc

```diff
@@ -3973,3 +3974,1055 @@ TEST_F(EnergyPlusFixture, ShadowCalculation_CSV)
         EXPECT_EQ(expected_values, stream_str);
     }
 }
+
+TEST_F(EnergyPlusFixture, SolarShadingTest_PolygonOverlap)
```

Comment:
Three new unit tests. 
`SolarShadingTest_PolygonOverlap` - tests similar to the 1Zone defect file with an overhang and a tree.
`SolarShadingTest_PolygonOverlap2` - tests two large south shades that both fully shade the wall and window, one behind the other. The expected results are simply the product of the transmittances.
`SolarShadingTest_PolygonOverlap3` - tests three large south shades, just to be sure the method works for more overlaps.

In case you're interested, here's the test failures with develop:
[PR10040-ShadingTransmittance-UnitTestFailureswithDevelop.txt](https://github.com/NREL/EnergyPlus/files/11680091/PR10040-ShadingTransmittance-UnitTestFailureswithDevelop.txt)